django/core/cache/backends/memcached.py

```diff
@@ -105,14 +105,14 @@ def close(self, **kwargs):
         self._cache.disconnect_all()
 
     def incr(self, key, delta=1, version=None):
-        # memcached doesn't support a negative delta
-        if delta < 0:
-            return self.decr(key, -delta, version=version)
         key = self.make_key(key, version=version)
         self.validate_key(key)
         try:
-            val = self._cache.incr(key, delta)
-
+            # memcached doesn't support a negative delta
+            if delta < 0:
```

Comment:
There is a minor behaviour change here. Previously calling `decr()` with `delta=0` would call `self._cache.decr()`, but now it'll call `self._cache.incr()` instead.

In theory this shouldn't be a problem, but am highlighting it.

Reply:
I don't think that's an issue.